grpc-sys/build.rs

```diff
@@ -67,6 +67,8 @@ mod imp {
         prepare_grpc();
 
         let dst = CMakeConfig::new("grpc")
+            .cflag("-w")
```

Comment:
BTW, here you're compiling grpc, not something in this repo. OK to ignore warnings.